### PR TITLE
PVS-50: Recipe Schema & Resolvers

### PR DESCRIPTION
**Pull request by arfusop-dev**
- [x] Add `logger` to `resolver` & `controller` for User, PatchNotes, & FAQs
- [x] Integrate external api spoonacular
- [x] Add recipe schema, typedefs
- [x] Add recipe resolvers: get all recipes (by user ID or on the system), get recipe by ID, create, edit, delete

### CODE REVIEW

File: gql/baseResolver.js [hunk 1]
```diff
@@ -23,28 +25,28 @@ class BaseResolver {
     this.schemas = { User }
 
     const baseErrors = {
```
Comment (KevinLaCarrubba) [inline gql/baseResolver.js:27]: would adding status codes to this be beneficial ?

File: gql/resolvers/patchNotes/types.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  type PatchNote {
+    _id: ID!
+    title: String!
+    description: String!
+    datePublished: Date
+    display: Boolean
+    targetDate: Date
+    targetVersion: Float
+    graphic: String
```
Comment (KevinLaCarrubba) [inline gql/resolvers/patchNotes/types.js:26]: we snappin pics ? sami's  wire frame gonna be on point

Comment (arfusop-dev) [reply gql/resolvers/patchNotes/types.js:26]: Possibly. Built it in incase we ever wanted to add snapshots etc

File: gql/baseResolver.js (hunk 1 above)
Comment (arfusop-dev) [reply gql/baseResolver.js:27]: Nope, discussed

**Pull request merged**